tenant-base/pom.xml

```diff
@@ -393,8 +393,10 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${maven-compiler-plugin.version}</version>
                 <configuration>
-                    <source>${target_jdk_version}</source>
-                    <target>${target_jdk_version}</target>
+                    <jdkToolchain>
```

Comment:
jdkToolchain is only valid when using maven-toolchains-plugin with toolchains.xml. Please remove.

Reply:
Thanks.